css/utility.css

```diff
@@ -0,0 +1,3 @@
+.hidden {
+  display: none !important ;
+}
```

Comment:
That is exactly what I meant 👍 
This is a file where so called "utility classes" are written. This usually means that these classes only do one styling. A great example is [tailwind](https://tailwindcss.com/), which is a utility classes based CSS framework. You could have a look at their utility classes 🙂